io/src/real_sense/sdk/real_sense_grabber.cpp

```diff
@@ -0,0 +1,316 @@
+/*
+ * Software License Agreement (BSD License)
+ *
+ *  Point Cloud Library (PCL) - www.pointclouds.org
+ *  Copyright (c) 2015-, Open Perception, Inc.
+ *  Copyright (c) 2016, Intel Corporation
```

Comment:
Sorry for adding unnecessary copyright in this file, the problem has been reverted.